gradle.properties

```diff
@@ -1,3 +0,0 @@
-org.gradle.daemon=true
```

Comment:
It auto file created. it compile without this.
you can keep it if you like.
what that mean gradle daemon?